lhotse/recipes/ami.py

```diff
@@ -0,0 +1,206 @@
+import logging
```

Comment:
Could you add a top-level docstring explaining what's the AMI corpus, as Dan suggested? It's fine to copy some existing description (e.g. from Kaldi or AMI download website). I'll make a separate PR adding these descriptions for LibriMix and LibriSpeech.

Reply:
Sure.